windows-agent/internal/proservices/landscape/executor.go

```diff
@@ -207,6 +207,11 @@ func (e executor) install(ctx context.Context, cmd *landscapeapi.Command_Install
 		if err != nil {
 			log.Warningf(ctx, "Landscape Install: failed to clean up %q after failed Install: %v", distro.Name(), err)
 		}
+
+		err = e.cloudInit().RemoveDistroData(distro.Name())
+		if err != nil {
+			log.Warningf(ctx, "Landscape install: failed to clean up %q cloud-init data after failed install: %v", distro.Name(), err)
+		}
```

Comment:
Since we don't really handle the errors, but log them in almost identical ways, I'd reorganize this code a little bit differently to get rid of the coverage warning:

```go
// Avoid error states by cleaning up on error
err := errors.Join(distro.Uninstall(ctx), e.cloudInit().RemoveDistroData(distro.Name()))
if err != nil {
			log.Warningf(ctx, "Landscape Install: failed to clean up %q after failed Install: %v", distro.Name(), err)
}
```

If you think it's too terse you can do

```go
e0 := distro.Uninstall(ctx)
e1 := e.cloudInit().RemoveDistroData(distro.Name()))
err := errors.Join(e0, e1)
if err != nil {
....
```

Either way communicates clearly that those actions don't have any dependency relationship between each other and we don't do much with their errors.